src/view/range.js

```diff
@@ -394,6 +394,17 @@ export default class Range {
 		}
 	}
 
+	/**
+	 * Checks whether given object is of `range` type following the convention set by
```

Comment:
This wording is a lot better. I complained in the other place that you link to `Node#is()` without much of an explanation. Saying that "following the convention set by" is a good way out.